config

```diff
@@ -18,3 +18,5 @@
 [pcuzner/vm2brick]
 [purpleidea/puppet-gluster]
 [spajus/libgfapi-ruby]
+[kshlm/gant]
+[kshlm/gogfapi]
```

Comment:
Trying to keep this alphabetical, so it'll be easier to locate stuff if/when the list gets long.

You ok to update this PR?